x-pack/solutions/observability/plugins/infra/public/pages/logs/page_content.tsx

```diff
@@ -96,21 +98,28 @@ export const LogsPageContent: React.FunctionComponent = () => {
       )}
 
       <Routes>
-        {routes.stream ? (
-          <Route path={routes.stream.path} component={StreamPage} />
-        ) : (
-          <Route
-            path="/stream"
-            exact
-            render={() => {
-              share.url.locators
-                .get<AllDatasetsLocatorParams>(ALL_DATASETS_LOCATOR_ID)
-                ?.navigate({});
-
-              return null;
-            }}
-          />
-        )}
+        <Route
```

Comment:
We still need to keep the old stream route available, because it can be enabled via an advanced setting. So I guess we only want to replace the `else` branch of the ternary?